### PR TITLE
Fixes #17236: Improve release notes for 6.1

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/17236

### CODE REVIEW

File: release-data/changelogs/6.1/index.adoc [hunk 1]
```diff
@@ -2,33 +2,51 @@
...
-* OpenSCAP: Runs OpenSCAP audits, collects and exposes them in the web interface
+Security-oriented features have been polished, with new operating systems supported in
+the CIS plugin, redesigned interface for the OpenSCAP plugin and better CVE fetching.
+
+=== Security features (through plugins)
+
+* CIS policy pack: Allows to apply pre-made CIS policies in one click using Rudder. Now supports
+  Debian 9 and Ubuntu 18.04 LTS in addition to RHEL/CentOS7.
+* CVE management: Reports CVEs affecting installed packages.
+* OpenSCAP: Runs OpenSCAP audits, collects and exposes them in the web interface. The Ui has been been
```
Comment (ncharles) [inline release-data/changelogs/6.1/index.adoc:18]: typo: UI

File: release-data/changelogs/6.1/index.adoc [hunk 2]
```diff
@@ -2,33 +2,51 @@
...
+hierarchic property system, extending node properties to
+global and group properties.
 
-* CIS policy pack: Allows to apply pre-made CIS policies in one click using Rudder
-* CVE management: Reports CVEs affecting installed packages
-* OpenSCAP: Runs OpenSCAP audits, collects and exposes them in the web interface
+Security-oriented features have been polished, with new operating systems supported in
+the CIS plugin, redesigned interface for the OpenSCAP plugin and better CVE fetching.
+
+=== Security features (through plugins)
```
Comment (ncharles) [inline release-data/changelogs/6.1/index.adoc:13]: isn't it a security feature not to have admin/admin anymore?

Comment (amousset) [reply release-data/changelogs/6.1/index.adoc:13]: Security features are feature allowing users to manages their systems securirty. Maybe the category name could be improved.

File: release-data/changelogs/6.1/index.adoc [hunk 3]
```diff
@@ -2,33 +2,51 @@
...
+  parameters. This allows powerful hierarchic variable definition, with easy
   overrides.
 
 === User experience improvements
 
-* The groups page has been redesigned with a much cleaner interface
+* The groups page has been redesigned with a much cleaner interface.
+* The resource manager in the technique editor has been improved.
+* New useful commands: `rudder server create-user` to create admin users,
+  `rudder agent policy-server` to manage a agent's policy server.
```
Comment (amousset) [inline release-data/changelogs/6.1/index.adoc:32]:
```suggestion
  `rudder agent policy-server` to manage an agent's policy server.
```

File: release-data/changelogs/6.1/index.adoc [hunk 4]
```diff
@@ -2,33 +2,51 @@
...
 
-* In addition to node properties, we added group properties, and improved global
-  parameters. This allows powerful hierarchized variable definition, with easy
+* In addition to existing node properties, we added group properties, and improved global
+  parameters. This allows powerful hierarchic variable definition, with easy
   overrides.
 
 === User experience improvements
 
-* The groups page has been redesigned with a much cleaner interface
```
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/index.adoc:19]: I would like to add the changes made on the technique editor (category can be selected at creation, you can add description to technique parameters, technique description is displayed as documentation on directive form as markdown)

File: release-data/changelogs/6.1/index.adoc (hunk 2 above)
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/index.adoc:13]:
```suggestion
=== IT Security oriented plugins
```
Could also be Cyber security oriented plugins

File: release-data/changelogs/6.1/index.adoc [hunk 5]
```diff
@@ -2,33 +2,51 @@
...
+* The groups page has been redesigned with a much cleaner interface.
+* The resource manager in the technique editor has been improved.
+* New useful commands: `rudder server create-user` to create admin users,
+  `rudder agent policy-server` to manage an agent's policy server.
```
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/index.adoc:32]:
```suggestion
**  `rudder agent policy-server` to manage an agent's policy server.  
```

File: release-data/changelogs/6.1/index.adoc [hunk 6]
```diff
@@ -2,33 +2,51 @@
...
+* In addition to existing node properties, we added group properties, and improved global
+  parameters. This allows powerful hierarchic variable definition, with easy
   overrides.
 
 === User experience improvements
 
-* The groups page has been redesigned with a much cleaner interface
+* The groups page has been redesigned with a much cleaner interface.
+* The resource manager in the technique editor has been improved.
+* New useful commands: `rudder server create-user` to create admin users,
```
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/index.adoc:31]:
```suggestion
* New useful commands:
** `rudder server create-user` to create admin users,
```

File: release-data/changelogs/6.1/index.adoc [hunk 7]
```diff
@@ -2,33 +2,51 @@
...
-  parameters. This allows powerful hierarchized variable definition, with easy
+* In addition to existing node properties, we added group properties, and improved global
+  parameters. This allows powerful hierarchic variable definition, with easy
   overrides.
 
 === User experience improvements
 
-* The groups page has been redesigned with a much cleaner interface
+* The groups page has been redesigned with a much cleaner interface.
+* The resource manager in the technique editor has been improved.
```
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/index.adoc:30]:
```suggestion
* Technique editor changes:
** Technique parameters can now be descripted, explaining their goal and share knowledge to your team
** Technique description is now displayed as markdown in Directives, so you can document how to use your technique with more details
** Create your techniques in a category different than "User techniques" 
** The resource manager in the technique editor has been improved.

```

File: release-data/changelogs/6.1/index.adoc (hunk 4 above)
Comment (amousset) [reply release-data/changelogs/6.1/index.adoc:19]: :+1:

**Comment by amousset** [comment]
Commit modified

**Pull request merged**